templates/debug.hbs

```diff
@@ -1,19 +1,42 @@
 <style>
     svg:first-child {
-        display: none;
+        display:none;
     }
-    svg {
+    code {
+        font-family:monospace;
+        font-size:11px;
+        color:#666;
+        display:block;
+    }
+    .wrap {
+        display: flex;
+        flex-wrap: wrap;
+    }
+    .item {
+        position:relative;
         display: inline-block;
-        border: 1px solid black;
+        margin:10px;
+        width: 120px;
+        height: 100px;
+        text-align:center;
+    }
+    svg.svg-item {
```

Comment:
Unfortunately this is still overqualified, no need to specify a tag and a class.

Reply:
![under-qualified](https://user-images.githubusercontent.com/1409408/34304680-095a621a-e73b-11e7-8cd7-1298e4649c19.png) 

mmm....?
`element:pseudo` = 11 vs `.my-class` = 10

Reply:
My bad, you are right! 
In the old template the SVGs weren't nested, which I had still in mind.

If we use the `u-hidden` utility class as you did in your previous version we could use a single class, which I'd prefer.

Sorry for making your first pull request so complicated. 😅

Reply:
no problemo. 
I'll push it later.